lib/generator.js

```diff
@@ -465,26 +425,29 @@ ScriptGenerator.prototype.generateDotNetDeploymentScript = function ScriptGenera
     return __this.writeDeploymentFiles(templateContent, __cb(_, __frame, 17, 2, _, true)); });};
 
 
-ScriptGenerator.prototype.generateAspNetCoreScript = function ScriptGenerator_prototype_generateAspNetCoreScript__15(templateFileName, options, _) { var fixedAspNetCoreProject, fixedRestore, fixedSolutionPath, lowerCaseScriptType, templateContent, __this = this; var __frame = { name: "ScriptGenerator_prototype_generateAspNetCoreScript__15", line: 468 }; return __func(_, this, arguments, ScriptGenerator_prototype_generateAspNetCoreScript__15, 2, __frame, function __$ScriptGenerator_prototype_generateAspNetCoreScript__15() {
+ScriptGenerator.prototype.generateAspNetCoreScript = function ScriptGenerator_prototype_generateAspNetCoreScript__15(templateFileName, options, _) { var prop, lowerCaseScriptType, templateContent, __this = this; var __frame = { name: "ScriptGenerator_prototype_generateAspNetCoreScript__15", line: 428 }; return __func(_, this, arguments, ScriptGenerator_prototype_generateAspNetCoreScript__15, 2, __frame, function __$ScriptGenerator_prototype_generateAspNetCoreScript__15() {
     argNotNull(templateFileName, "templateFileName");
 
-    fixedAspNetCoreProject = ((__this.scriptType === ScriptType.batch) ? fixPathSeperatorToWindows(options.aspNetCoreProject) : fixPathSeperatorToUnix(options.aspNetCoreProject));
-    fixedRestore = ((__this.scriptType === ScriptType.batch) ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore));
-    fixedSolutionPath = ((__this.scriptType === ScriptType.batch) ? fixPathSeperatorToWindows(options.solutionPath) : fixPathSeperatorToUnix(options.solutionPath));
+    if ((__this.scriptType === scriptType.bash)) {
+      for (prop in options) {
+        options[prop] = fixPathSeperatorToUnix(options[prop]); }; }
 
-    lowerCaseScriptType = __this.scriptType.toLowerCase();
+     else {
+      for (prop in options) {
+        options[prop] = fixPathSeperatorToWindows(options[prop]); }; } ;
```

Comment:
typo 'Seperator' -> 'Separator'. I know it was already wrong before, but may as well fix :)